testing/integration/README.md

```diff
@@ -19,3 +19,11 @@ The following flow has been defined as a default usage:
 1. Wasm bytecode is now expected to be received through a binary type (`&[u8]`). This be upgraded to work Rust module compiled
 at test time.
 2. Some testing and examples should be added to demonstrate how the framework works.
+
+TODO: (hack to get coverage reports from actors + integration tests) 
+```shell
+cargo build -p "*actor"
+export SKIP_WASM_BUILD true
+export FVM_STORE_ARTIFACT_DIR ../../target/llvm-cov-target/
+cargo llvm-cov -p fvm_integration_tests --lcov
```

Comment:
Let's stick with bash.

Reply:
I meant `export FOO bar` isn't valid bash. `export FOO=bar` is.

Reply:
ah, well who uses bash anyways, fish is better :^)